adapter/thing_management.md

```diff
@@ -0,0 +1,404 @@
+# Thing Management
+
+The following document describes the interaction flows and FIMP specification used in management of things.
+
+## Flows
+
+### Network Inclusion
+
+Network inclusion flow is used by wireless network adapters, such as Z-Wave and Zigbee, to include new devices into the network and add them to the system.
+
+* Client sends `cmd.thing.inclusion` command to the adapter with `true` value to start inclusion process and switch adapter into pairing mode.
+* User interacts with a physical device to switch it into pairing mode, e.g. clicks a button on a device three times in a quick sequence.
```

Comment:
Maybe it would add too much of not needed information but I would write that info of how to put device into pairing mode should be described in device's manual. So e.g. "... switch it into pairing mode. That process should be described in device's manual". 

But maybe this additional info is not needed.

Reply:
Good point. Moar is always moar.